### DISCUSSION TITLE
Attacking object-detection yolo model.

### DISCUSSION THREAD

**Discussion by dalqattan** [Trusted-AI/adversarial-robustness-toolbox#2202] [Q&A]
Hi

 I would like to run the available attacks (Dpatch, RobustDpatch, Shapeshifter) for object detection on a yolo model.

Is there any example?

**Comment by Foxglove144** [comment]
[This ](https://github.com/Trusted-AI/adversarial-robustness-toolbox/blob/main/examples/get_started_yolo.py)is an example script for generating adversarial examples on pretrained YOLOv3/YOLOv5 model(s), and [here ](https://github.com/Trusted-AI/adversarial-robustness-toolbox/blob/main/examples/get_started_fasterrcnn.py)is an example of generating a RobustDPatch effective against FasterRCNN. You can easily replace the FasterRCNN detector with a YOLO detector in the second example.

**Reply by dalqattan** [reply]
Thanks, Is YOLO8 supported?

**Reply by Foxglove144** [reply]
AFAIK, no it is not.
